app/src/main/res/values/strings.xml

```diff
@@ -1,81 +1,179 @@
+         console output to the Android logcat. -->
     <string name="developer_options_show_console">Redirect Console to Logcat</string>
+    <!-- LOCALIZATION NOTE (developer_options_env_override): The string 'developer_options_env_override'
+         labels an ON/OFF switch in the developer options dialog and is used to toggle loading externally
+         generated environments and overriding the ones built in. -->
     <string name="developer_options_env_override">Enable Environment Override</string>
+    <!-- LOCALIZATION NOTE (developer_options_multiprocess): The string 'developer_options_multiprocess'
+         labels an ON/OFF switch in the developer options dialog and is used to toggle running content
+         in a separate process. -->
     <string name="developer_options_multiprocess">Enable Multiprocess</string>
+    <!-- LOCALIZATION NOTE (developer_options_ua_mode): The string 'developer_options_ua_mode' is
+         used to label a set of radio buttons that allow the user to change the user agent (UA) string
+         of the browser. -->
     <string name="developer_options_ua_mode">User Agent Mode</string>
+    <!-- LOCALIZATION NOTE (developer_options_events): FIXME This option should be removed -->
     <string name="developer_options_events">Input Events</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_display_density">Display Density:</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_window_size">Window Size:</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_mouse">Mouse</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_touch">Touch</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_edit">Edit</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_save">Save</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_display_dpi">Display DPI:</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_max_window_size">Max. Window Size:</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_msaa">MSAA</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_msaa_disabled">Disabled</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_msaa_2">2x</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_msaa_4">4x</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_ua_desktop">Desktop</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_ua_mobile">Mobile</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_ua_vr">VR</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_reset">Reset Developer Settings</string>
+    <!-- LOCALIZATION NOTE ():  -->
```

Comment:
I know it's missing/WIP, but I'd add a note about this `Reset` relating to resetting developer options